pkg/volume/iscsi/iscsi_util.go

```diff
@@ -205,6 +205,16 @@ func (util *ISCSIUtil) AttachDisk(b iscsiDiskMounter) error {
 	iscsiTransport = extractTransportname(string(out))
 
 	bkpPortal := b.Portals
+
+	// create new iface and copy parameters pre-configured iface to the created iface
+	if b.InitiatorName != "" {
+		err = cloneIface(b, bkpPortal[0])
+		if err == nil {
+			// update iface name with <target portal>:<volume name>
+			b.Iface = bkpPortal[0] + ":" + b.volName
```

Comment:
OK, so the iscsiadm doesn't accept over 64 characters. We should check length of portal address + volume name.

```
root# sudo iscsiadm -m iface --op=new -I 12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890
Invalid iface name 12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890. Must be from 1 to 64 characters.
```